src/mam4xx/mo_chm_diags.hpp

```diff
@@ -314,7 +314,7 @@ void chm_diags(
   // accumulators
   // if (history_aerosol.and..not .history_verbose) then
 
-  for (int nn = lchnk; nn < pcnst; nn++) {
+  for (int nn = lchnk; nn < gas_pcnst; nn++) {
```

Comment:
I don't quite understand this change. Can we verify if it is okay to change it to gas_pcnst?
If it takes time, let's just revert this change and investigate it later.

Reply:
Let's create a GitHub issue for this and investigate. We can revert this change for now. We will fix it after we understand the right way to do it.